scripts/install

```diff
@@ -39,6 +40,11 @@ test ! -e "$final_path" || ynh_die --message="This path already contains a folde
 # Register (book) web path
 ynh_webpath_register --app=$app --domain=$domain --path_url=$path_url
 
+if [[ "$architecture" -ne "amd64" ]]
```

Comment:
```suggestion
if [[ "$architecture" != "amd64" ]]
```

"Fun" fact ... `-ne` and `-eq` are only for comparing numbers ... Comparing string is done using `==` and `!=`:

```bash
$ [[ "foo" != "bar" ]] && echo true || echo false
true
$ [[ "foo" -ne "bar" ]] && echo true || echo false
false # <- uhoh
$ [[ "foo" -eq "bar" ]] && echo true || echo false
true  # <- go home, bash, u drunk
```